modules/RTC/RTCUtils.js

```diff
@@ -1101,7 +1101,15 @@ class RTCUtils extends Listenable {
             || RTCBrowserType.isNWJS()
             || RTCBrowserType.isElectron()
             || RTCBrowserType.isReactNative()) {
-            this.pcConstraints = { optional: [] };
+            this.pcConstraints = { optional: [
```

Comment:
maybe we should do an a/b test for these changes?

Reply:
PSNR tests? And running with it for a week and checking overall callstats quality?

Reply:
yeah i was thinking of doing something like what boris did for the video suspension change